configs/components/fesom/fesom-2.1.yaml

```diff
@@ -201,52 +200,47 @@ restart_out_sources:
         fesom_bin_restart_info: fesom_bin_restart/*.info
         fesom_bin_restart: fesom_bin_restart/np${nproc}/*
         wiso_restart: fesom.${end_date!syear}.wiso.restart.nc
-        icb_restart: iceberg.restart #.${parent_date!syear}
-        icb_restart_ISM: iceberg.restart.ISM
+        icb_restart: iceberg.restart.${end_date!syear} #_${start_date!syear!smonth!sday}-${end_date!syear!smonth!sday} #.${parent_date!syear}
+        icb_restart_ISM: iceberg.restart.ISM.${end_date!syear} #_${start_date!syear!smonth!sday}-${end_date!syear!smonth!sday}
 
 outdata_sources:
         fesom: "*.fesom.*.nc"
 outdata_targets:
-        fesom: "*.fesom.*${start_date!smonth}.${start_date!sday}.nc"
+        #fesom: "*.fesom.*${start_date!smonth}.${start_date!sday}.nc"
+        fesom: "*.fesom.*.nc"
 
 choose_use_icebergs:
         True:
-
             add_restart_in_files:
                     icb_restart: icb_restart
                     icb_restart_ISM: icb_restart_ISM
-
-            add_restart_out_files:
+            restart_out_files:
                     icb_restart: icb_restart
                     icb_restart_ISM: icb_restart_ISM
-
+            #restart_in_intermediate:
+            #        icb_restart: iceberg.restart.${parent_date!syear} #${start_date!smonth!sday}-${parent_date!syear}${end_date!smonth!sday}
+            #        icb_restart_ISM: iceberg.restart.ISM.${parent_date!syear} #${start_date!smonth!sday}-${parent_date!syear}${end_date!smonth!sday}
+            #restart_in_targets:
+            #        icb_restart: iceberg.restart.${parent_date!syear} #!smonth!sday}-${end_date!syear!smonth!sday}
+            #        icb_restart_ISM: iceberg.restart.ISM.${parent_date!syear} #!smonth!sday}-${end_date!syear!smonth!sday}
+            #restart_out_targets:
+            #        icb_restart: iceberg.restart.${start_date!syear} #!smonth!sday}-${end_date!syear!smonth!sday}
+            #        icb_restart_ISM: iceberg.restart.ISM.${start_date!syear} #!smonth!sday}-${end_date!syear!smonth!sday}
+            
```

Comment:
you can remove these lines since they are outside of the choose block already :)